spec/controllers/books_controller_spec.rb

```diff
@@ -1,12 +1,56 @@
 require 'rails_helper'
 
 RSpec.describe BooksController, type: :controller do
-
   describe "GET #index" do
     it "returns http success" do
       get :index
       expect(response).to have_http_status(:success)
     end
-  end
 
+    context "no books" do
+      it "returns empty array" do
+        get :index
+
+        expect(JSON.parse(response.body)).to eq []
+      end
+    end
+
+    context "book exists" do
+      it "returns array containing all book" do
```

Comment:
`it "returns array containing all book"`  says array but we are expecting count

Reply:
Franceska got the point though. We should follow domain language instead of describe particular implementation.
I'd go with `it "returns list of books"`